### ISSUE TITLE
Voting question

### ISSUE THREAD

**Issue by 1525898173** [xhanxu/Mamba3D#20]
Voting question: Why is it that when I train a model from scratch or fine-tune it, the accuracy I get, for example, 93.2%, is expected to be the same when performing voting? In the code, the model is tested again before the voting process begins. Logically, the accuracy here should be the same as 93.2%, but when using your model for voting, the accuracy is different — sometimes higher and sometimes lower. What is the reason for this?

2025-02-27 15:55:41,696 - ModelNet - INFO - The size of test data is 2468
2025-02-27 15:55:41,696 - ModelNet - INFO - Load processed data from /root/autodl-tmp/modelnet40_normal_resampled/modelnet40_test_8192pts_fps.dat...
2025-02-27 15:55:42,155 - finetune_modelnet - INFO - Loading weights from /root/autodl-tmp/CPoint/experiments/finetune_modelnet/cfgs/PointMamba_modelnet_scratch_test/ckpt-best.pth...
2025-02-27 15:55:42,256 - finetune_modelnet - INFO - ckpts @ 130 epoch( performance = {'acc': tensor(93.2196)})
2025-02-27 15:55:49,741 - finetune_modelnet - INFO - [TEST] acc = 93.4222
2025-02-27 15:55:49,741 - finetune_modelnet - INFO - [GPU Mem] MEM
